### ISSUE TITLE
Suggestion: Rename the operator from `validation/main/X` to `val/main/X`

### ISSUE THREAD

**Issue by 23pointsNorth** [chainer/chainer#7591]
Recent models rely heavily on multiple loss functions. If we want to track them with a `PrintReport`, the width of the line heading becomes far too wide, containing mostly empty space, mainly resulting from the `validation` tag in front of the loss.

Could there be a consensus on possible abbreviation?

e.g.
```
epoch       iteration   main/loss   main/mae    main/gnll   main/weighted  main/VAE    main/VAE_REC  main/VAE_KL  validation/main/loss  validation/main/mae  validation/main/weighted  validation/main/VAE  elapsed_
time
1           356         31.2785     1.39209     3.45184     27.4938        0.332836    32933.7       349.925      26.5542               1.27629              23.1902                   0.32403              114.618 
      
```

to 
```
epoch       iteration   main/loss   main/mae    main/gnll   main/weighted  main/VAE    main/VAE_REC  main/VAE_KL  val/main/loss  val/main/mae  val/main/weighted  val/main/VAE  elapsed_time
1           356         31.2785     1.39209     3.45184     27.4938        0.332836    32933.7       349.925      26.5542        1.27629       23.1902            0.32403       114.618 
      
```

We can even decrease the whitespace around the iteration/epoch columns to be `numer_of_digits(max_epoch)+2`. Alternatively, decrease the [default width of 10](https://github.com/chainer/chainer/blob/ac1f90a442f0481fd6fed3618b48518529713a9b/chainer/training/extensions/print_report.py#L34) to something smaller - 6 or 8.

**Comment by yordanh** [comment]
What is the reason behind the convention of having main in the names. Appears everywhere and takes a good amount of column space. At least when printing can't it simply be train/<loss_name> and val<loss_name>.

**Comment by crcrpar** [comment]
I guess it's related to how `chainer.training.Updater` and its child classes manage optimizer(s) & iterator(s).
 
For example,[`chainer.training.StandardUpdater`](https://docs.chainer.org/en/stable/reference/generated/chainer.training.updaters.StandardUpdater.html#chainer.training.updaters.StandardUpdater) internally manages optimizer(s) & iterator(s) using dictionaries as follows.
https://github.com/chainer/chainer/blob/d297a1787ebb35cfce279dde0df64dd2feba4327/chainer/training/updaters/standard_updater.py#L67-L79

In general, I think reasonable both to shorten validation to val and eliminating main from print :)

**Comment by 23pointsNorth** [comment]
Indeed. I've been trying to find where the `validation` string is used to prefix the optimizer/iterator similar to the dictionary created for `main`, but couldn't really locate it in the code. (A repo wide search didn't help, just showed all the examples that use PrintReport)

What would the best way to implement your suggestions be in regards to shortening and eliminating?

**Comment by crcrpar** [comment]
> where the `validation` string is used to prefix the optimizer/iterator

[docs](https://docs.chainer.org/en/stable/reference/generated/chainer.report.html?highlight=reporter) seems to be the answer.

> If these values are reported inside the Evaluator extension, 'validation/' is added at the head of the link name, thus the item names are changed to 'validation/main/abs_error' and 'validation/main/squared_error' ('validation' is the default name of the Evaluator extension).

**Comment by kmaehashi** [comment]
@23pointsNorth `validation` is the default name of the `Evaluator` extension.
https://docs.chainer.org/en/stable/reference/generated/chainer.training.extensions.Evaluator.html?#chainer.training.extensions.Evaluator.default_name
https://github.com/chainer/chainer/blob/v6.1.0/examples/mnist/train_mnist.py#L113-L120

You can give it a different name like this:

```py
trainer.extend(extensions.Evaluator(iter, model, device=device), name='val')
```
